compiler/rustc_trait_selection/src/traits/query/normalize.rs

```diff
@@ -162,6 +163,40 @@ struct QueryNormalizer<'cx, 'tcx> {
     universes: Vec<Option<ty::UniverseIndex>>,
 }
 
+impl<'cx, 'tcx> QueryNormalizer<'cx, 'tcx> {
+    fn fold_reveal(
```

Comment:
Same thing here as in the other `fold_reveal` place, the name should be more descriptive.